Adding doc on Customizing ROSA Console URL

FungCharlotte opened the pull request:
Wrote new documentation on how to customize ROSA Consoel URL

FungCharlotte committed:
added new ROSA documentation

Signed-off-by: Charlotte Fung <ewofung@yahoo.fr>

thatcherhubbard committed:
Update index.md

Adding a bit of formatting for user commands.

FungCharlotte committed:
Merge pull request #6 from thatcherhubbard/patch-1

Update index.md

paulczar commented on content/rosa/customizing-console-route/index.md:
````diff
@@ -0,0 +1,123 @@
+    > Skip this step if you already have wildcard policy enabled on your cluster
+
+     ```bash
+     rosa edit ingress -c <cluster_name> <default-ingress_id> --wildcard-policy WildcardsAllowed
````
why? I don't see anything in this doc that would require a wildcard route ?

paulczar commented on content/rosa/customizing-console-route/index.md:
````diff
@@ -0,0 +1,123 @@
+   
+     ```bash
+     export SCRATCH_DIR=/tmp/scratch
+     export ZONE_ID=$(aws route53 list-hosted-zones-by-name --output json \
````
where is this variable being used ?

FungCharlotte replied:
Oops, I will remove the ZONE_ID variable. I don't use that any where in the doc

paulczar commented on content/rosa/customizing-console-route/index.md:
```diff
@@ -0,0 +1,123 @@
+
+
+### Test
+1. Retrieve ROSA console URL
```
should we have them log out and log back in ?

FungCharlotte replied:
The assumption is that they haven't logged in already, but if they have, then they have to logout and back in. Will add that to the doc. Thank you

FungCharlotte committed:
Update index.md

removed unnecessary variable, added instruction to log out and log back in for console url test, and specified this was tested on ROSA classic cluster only (not HCP)
